src/site/stages/build/process-cms-exports/transformers/node-press_release.js

```diff
@@ -27,7 +27,11 @@ const transform = entity => ({
     entity.fieldOffice && entity.fieldOffice[0]
       ? { entity: entity.fieldOffice[0] }
       : null,
-  fieldPdfVersion: entity.fieldPdfVersion[0] || null,
+  fieldPdfVersion: entity.fieldPdfVersion[0]
+    ? {
+        entity: entity.fieldPdfVersion[0],
+      }
+    : null,
```

Comment:
I wanted to suggest a more elegant way to do this but after reading it a few times I realized that the technology doesn't exist right now

Reply:
I added a helper function.